ci/test-coverage.sh

```diff
@@ -1,28 +1,33 @@
 #!/usr/bin/env bash
+
 set -e
-cd "$(dirname "$0")/.."
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+
+if [[ -z $CI ]]; then
+  echo "This script is used by CI environment. Use scripts/coverage.sh directly if you only want to obtain the coverage report"
```

Comment:
nit: seems this line is over 100 chars.

silly way to word wrap:

```
echo "This ..."
echo "Use ..."
```

or just use `\`. ;)

Reply:
fixed: [4e266d8](https://github.com/anza-xyz/agave/pull/1260/commits/4e266d8bbd4c9d18f50b2a27781c363832dbb6cd)